src/almanack/metrics/data.py

```diff
@@ -477,6 +488,8 @@ def compute_repo_data(repo_path: str) -> None:
             if doi_citation_data["publication_date"] is not None
             else None
         ),
+        # placeholder for sustainability score
+        "repo-almanack-sustainability-score": 0,
```

Comment:
Thanks, see also comments in https://github.com/software-gardening/almanack/pull/208#discussion_r1894632248 . I like the idea of using NA or None here to avoid confusion about the information provided. I'll make some changes here and follow up.